config/i18n.json

```diff
@@ -998,7 +998,9 @@
     "RecordBooks": "Record Books"
   },
   "Records": {
-    "Title": "Records"
+    "Title": "Records",
+    "UniversalOrnaments": "Universal Ornaments",
```

Comment:
3655910122 2497860177 or 466446949 have a pre-localized and bungie-standard copy of this string